templates/HOWTO.postgresql.conf

```diff
@@ -2,11 +2,13 @@
 How to add a new PostgreSQL version
 ===================================
 
-1) Download the Debian package 'postgresql-9.X_[...].deb' from
+1) Download the Debian package 'postgresql-<version>_[...].deb' from
 http://apt.postgresql.org/pub/repos/apt/pool/main/p/postgresql-9.X/
 
-2) Extract the 'usr/share/postgresql/9.1/postgresql.conf.sample' file
-and save it under the 'templates' role directory
+2) Extract the 'usr/share/postgresql/9.1/postgresql.conf.sample' file with
+   dpkg-deb -x postgresql-<version>_[...].deb dir/to/extract/to/
+
+   and save it under the 'templates' role directory
     => templates/postgresql.conf.9.{X}.orig
```

Comment:
Update to `<version>` here too... but should we match the role and use `<postgresql_version>`?

Reply:
Done.